bokeh/core/_templates/doc_js.js

```diff
@@ -1,4 +1,6 @@
 var docs_json = {{ docs_json }};
 var render_items = {{ render_items }};
 
-Bokeh.embed.embed_items(docs_json, render_items{%- if websocket_url -%}, "{{ websocket_url }}" {%- endif -%});
+Bokeh.$(function() {
+    Bokeh.embed.embed_items(docs_json, render_items{%- if websocket_url -%}, "{{ websocket_url }}" {%- endif -%});
+})
```

Comment:
no newline or semicolon I can imagine breaking something (say we concat another thing after this).